Sources/Brave/WebFilters/ResourceDownloader.swift

```diff
@@ -265,14 +175,15 @@ actor ResourceDownloader: Sendable {
     }
   }
   
-  /// Get an existing etag for the given `Resource`
+  /// Get a creation date for the file downloaded for the given `Resource`.
+  /// If the file is not downloaded a nil is returned.
   static func creationDate(for resource: Resource) throws -> Date? {
     guard let fileURL = downloadedFileURL(for: resource) else { return nil }
```

Comment:
Why is this method static? Why wouldn't it be a method on `Resource`?

Reply:
I could move it but now I'll be cleaning up things I did way back when. Can I make a small pr moving these methods after this?

Reply:
At this point this PR is like 75% cleanup 25% add custom filter lists, so may as well do it here

Reply:
ok, moved

Reply:
dbb03c7b4658a9cd278157d0425c0b988de7f9de